lib/amaysim_logger.rb

```diff
@@ -48,35 +48,46 @@ def logger
     delegate :level, :level=, to: :logger
     delegate :formatter, :formatter=, to: :logger
     delegate :info?, :debug?, :warn?, :error?, :unknown?, to: :logger
+    delegate :<<, to: :logger
```

Comment:
I don't think this will work. you need to handle the this as you handle the other log methods